ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/svarinn/SvarInnMessage.java

```diff
@@ -0,0 +1,89 @@
+package no.difi.meldingsutveksling.ks.svarinn;
+
+import lombok.Data;
+import lombok.NonNull;
+import no.difi.meldingsutveksling.core.EDUCore;
+import no.difi.meldingsutveksling.core.Receiver;
+import no.difi.meldingsutveksling.core.Sender;
+import no.difi.meldingsutveksling.noarkexchange.receive.PayloadConverter;
+import no.difi.meldingsutveksling.noarkexchange.receive.PayloadConverterImpl;
+import no.difi.meldingsutveksling.noarkexchange.schema.core.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Data
+public class SvarInnMessage {
+    @NonNull
+    Forsendelse forsendelse;
+    @NonNull
+    List<SvarInnFile> svarInnFiles;
+
+    private PayloadConverter<MeldingType> payloadConverter = new PayloadConverterImpl<>(MeldingType.class, "http://www.arkivverket.no/Noark4-1-WS-WD/types", MeldingType.class.getName());
+
+    EDUCore toEduCore() {
```

Comment:
Var litt inne på tanken, men det er litt for tett kobling med FIKS Forsendelse som jeg tenkte er best å ha i FIKS modulen